arangod/Aql/IResearchViewNode.h

```diff
@@ -219,6 +236,11 @@ class IResearchViewNode final : public arangodb::aql::ExecutionNode {
 
   /// @brief IResearchViewNode options
   Options _options;
+
+  /// @brief prototype collection for enabling smart joins
+  aql::Collection const* _prototypeCollection;
```

Comment:
BTW what is the purpose of having these variables as `IResearchViewNode` members?